### PR TITLE
XWIKI-10473: Add 'Move' to Page actions submenu

### PR DESCRIPTION
**Pull request by msmeria**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/menus_content.vm [hunk 1]
```diff
@@ -199,13 +199,13 @@
...
   #if ($canCopy)
-    #set ($discard = $staticExtensions.add( { 'content': "#submenuitem($xwiki.getURL($doc.documentReference, 'view', 'xpage=copy') $services.localization.render('core.menu.copy') 'tmActionCopy', '' 'share')", 'order': 20000, 'separator': true}))
+    #set ($discard = $staticExtensions.add( { 'content': "#submenuitem($xwiki.getURL($doc.documentReference, 'view', 'xpage=copy') $services.localization.render('core.menu.copy') 'tmActionCopy', '' 'share')", 'order': 30000}))
   #end
   ##
   ## DELETE / RENAME
   ##
   #if ($canDelete)
-    #set ($discard = $staticExtensions.add( { 'content': "#submenuitem($xwiki.getURL($doc.documentReference, 'view', 'xpage=rename&amp;step=1') $services.localization.render('core.menu.rename') 'tmActionRename', '' 'edit')", 'order': 30000}))
+    #set ($discard = $staticExtensions.add( { 'content': "#submenuitem($xwiki.getURL($doc.documentReference, 'view', 'xpage=rename&amp;step=1') $services.localization.render('core.menu.rename') 'tmActionRename', '' 'edit')", 'order': 20000, 'separator': true}))
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/menus_content.vm]: Any particular reason for moving the separator? Why separate Rename from Copy? Right now Copy, Rename and Delete are grouped.

Comment (msmeria) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/menus_content.vm]: I did not move the separator. I just switched the places for Copy and Move. Look at the image I've added to http://jira.xwiki.org/browse/XWIKI-10473 for the result of my modification. I did this at @evalica's recommendation.

Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/menus_content.vm]: I'm not sure about the switch. I would keep the less dangerous operation first, which is Copy, and the most dangerous operation last, which is Delete. Rename/Move is in between.

Comment (evalica) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/menus_content.vm]: In the user's opinion "Rename" should not be such a dangerous operation. 

I've asked msmeria to switch the order for visual reasons: having longer labels together, while keeping the visual flow go down. 

Having --, -, --, - entries doesn't not provide a nice visual transition.

**Pull request merged**